frontend/src/metabase/public/components/EmbedModal/SelectEmbedTypePane/SelectEmbedTypePane.unit.spec.tsx

```diff
@@ -51,7 +51,7 @@ const setup = ({
         currentUser: createMockUser({ is_superuser: isAdmin }),
         settings: mockSettings({
           "enable-public-sharing": isPublicSharingEnabled,
-          "enable-embedding": isApplicationEmbeddingEnabled,
+          "enable-embedding-static": isStaticEmbeddingEnabled,
```

Comment:
the static embedding option is now controlled by its own embedding flag not `enable-embedding` (old one)